config/default.yml

```diff
@@ -77,6 +77,12 @@ Minitest/GlobalExpectations:
   Enabled: true
   VersionAdded: '0.7'
 
+Minitest/MultipleAssertions:
+  Description: 'This cop checks if test cases contain too many assertion calls.'
+  Enabled: 'pending'
+  VersionAdded: '0.10'
+  Max: 3
```

Comment:
Is `3` the default max value when this cop is enabled? We should mention it in the documentation somewhere. Maybe here: https://github.com/rubocop-hq/rubocop-minitest/pull/88/files#diff-579fb7ac8b2c940a3dfbb36ea97f6ea1R394

Reply:
This will appear in the auto-generated docs (the default value), but extending the cop description is never a bad idea.